### ISSUE TITLE
create sample on absolute track number behavior

### ISSUE THREAD

**Issue by dericed** [mipops/dvrescue#217]
To create the samples:
- recording a very short recording near the beginning of a blank dv tape
- then fast forward past blank parts, then record another short part
- then go near end of tape and record another short part
- then capture all recordings

If possible include both SP and LP recordings.
If possible perform the test on more than one deck.

Label all files with SP or LP, recording deck name, and approximate position on the tape.

**Comment by libbyhopfauf** [comment]
Just to clarify, should the same tape include both LP and SP recordings or separate tapes? And the files should be split up by tape position vs. the full recording?

**Comment by dericed** [comment]
I don't think it matters. My hypothesis is that SP and LP recordings use a different incrementation pattern in absolute track number. The files could be captured separately, as there would likely be a lot of blank tape between recorded sections. But if its one capture of a tape with large blank parts that's all fine too.

**Comment by dericed** [comment]
Updating notes that LP seems to cause recycling of abst values, but am not clear since the only known sample i have of LP is DVR007 which is also severely damaged.

**Comment by libbyhopfauf** [comment]
> Updating notes that LP seems to cause recycling of abst values, but am not clear since the only known sample i have of LP is DVR007 which is also severely damaged.

Paul’s making more samples. Should have those to you by the end of the week.

**Comment by dericed** [comment]
samples by Paul at https://drive.google.com/drive/folders/1S4lCxdCJXb_9hhYQfH96HRakjdg43G_M. This needs a readme or some documentation though to make some conclusions.

**Comment by dericed** [comment]
@libbyhopfauf perhaps we should have a samples doc in the documentation that points to known files that show various issues.
